src/ansys/acp/core/_tree_objects/_mesh_data.py

```diff
@@ -88,7 +86,10 @@ def _expand_array(
     target_array = np.ones(target_shape, dtype=np.float64) * np.nan
     for idx, (label, value) in enumerate(zip(labels.data_labels, array)):
         if idx % culling_factor == 0:
-            target_array[labels.mesh_label_to_index_map[label]] = value
+            try:
```

Comment:
Allows plotting on a mesh that does not contain all labels present in the data. Maybe this could be refined further, but it allows e.g. plotting the analysis ply thickness only on the solid mesh.